src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Bulldozer bulldozer = new Bulldozer();
+        Excavator excavator = new Excavator();
+        Truck truck = new Truck();
+
+        bulldozer.doWork();
+        excavator.doWork();
+        truck.doWork();
+        bulldozer.stopWork();
+        excavator.stopWork();
+        truck.stopWork();
```

Comment:
[Checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop#use-abstract-reference-instead-of-specific-one-where-possible): Use abstract reference instead of specific one where possible. И давай создадим массив из машин и будет запускать их и останавливать в цикле

Reply:
Поправил